discovery-infra/test_infra/helper_classes/kube_helpers/secret.py

```diff
@@ -56,12 +51,10 @@ def get(self) -> dict:
 
 
 def deploy_default_secret(
-    kube_api_client: ApiClient, name: str, ignore_conflict: bool = True, pull_secret: Optional[str] = None
+    kube_api_client: ApiClient, name: str, namespace: str, pull_secret: str, ignore_conflict: bool = True
 ) -> Secret:
-    if pull_secret is None:
-        pull_secret = os.environ.get("PULL_SECRET", "")
```

Comment:
What about the pull secret?

Reply:
We are getting it in the function arguments

Reply:
Missed that. :)